lib/internal/streams/destroy.js

```diff
@@ -1,4 +1,8 @@
-'use strict'
+/* replacement start */
+const process = require('process')
+/* replacement end */
+
+;('use strict')
```

Comment:
Can't you put require after the `use strict`?
This applies to this and all other similar files.

Reply:
This is generated by the [headers.mjs](https://github.com/nodejs/readable-stream/blob/main/build/headers.mjs) build script. I agree that it is not very pretty, but there is existing code with the same patter: [test-stream-writable-destroy.js](https://github.com/nodejs/readable-stream/blob/main/test/parallel/test-stream-writable-destroy.js)

Reply:
You could resolve this by replacing line 104 of `header.mjs` with this:

```javascript
if (content.match(strictMatcher)) {
  content = content.replace(strictMatcher, `$&;${header}`)
} else {
  content = header + content
}
```

The else branch was the original line.

Where: 

`const strictMatcher = /^(['"']use strict.+)/` 

is added below line 17.

Reply:
@ShogunPanda thanks! I changed the build code. Now the patched code looks much prettier.